internal/expr/bindtypes.go

```diff
@@ -259,6 +259,44 @@ func (e *columnsInsertExpr) bindTypes(argInfo typeinfo.ArgInfo) (tie any, err er
 	return &typedInsertExpr{insertColumns: cols}, nil
 }
 
+// basicInsertExpr is an input expression occurring within an INSERT statement
+// that consists of columns on the left and type accessors or literals on the
+// right. Unlike the columnInsertExpr, the values on the right are independent
+// of the columns on the left and are matched by position rather than by name.
+// e.g. (col1, col2, col3) VALUES ($M.key, "literal value", $T.value).
+type basicInsertExpr struct {
+	columns []columnAccessor
+	sources []valueAccessor
+	raw     string
+}
+
+// String returns a text representation for debugging and testing purposes.
+func (e *basicInsertExpr) String() string {
+	return fmt.Sprintf("RenamingInsert[%v %v]", e.columns, e.sources)
```

Comment:
Maybe it would be useful to use commas to separate the sources, if you have an expression like `1 + 2 + 3` or even two words it might look like two separate sources instead of one.

Reply:
True, its a bit confusing, but this is only for our tests and will make the diff even worse for this PR.